Microsoft.DotNet.Interactive.PowerShell/Microsoft.DotNet.Interactive.PowerShell.csproj

```diff
@@ -18,76 +18,43 @@
     <NoWarn>;NU5104;NU5100;2003;8002</NoWarn>
   </PropertyGroup>
 
-  <!-- PowerShell Module Versions -->
-  <PropertyGroup>
-    <PackageManagementVersion>1.4.6</PackageManagementVersion>
-    <PowerShellGetVersion>2.2.3</PowerShellGetVersion>
-    <MicrosoftPowerShellArchiveVersion>1.2.5</MicrosoftPowerShellArchiveVersion>
-    <ThreadJobVersion>2.0.3</ThreadJobVersion>
-  </PropertyGroup>
-
   <!-- PowerShell Module Package References -->
   <ItemGroup>
-    <PackageReference Include="PackageManagement" Version="$(PackageManagementVersion)">
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="PowerShellGet" Version="$(PowerShellGetVersion)">
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Microsoft.PowerShell.Archive" Version="$(MicrosoftPowerShellArchiveVersion)">
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="ThreadJob" Version="$(ThreadJobVersion)">
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
+    <PackageReference Include="PackageManagement" Version="1.4.6" GeneratePathProperty="true" />
```

Comment:
Why do away with `<PrivateAssets>all</PrivateAssets>`/

Reply:
Because that wasn't copying the files where we need, which is under the `Modules/` directory.

Reply:
NM, I misunderstood what PrivateAssets was doing here.  Will restore in a future PR.